ProjFS.Mac/PrjFSKext/KauthHandler.cpp

```diff
@@ -808,10 +808,11 @@ KEXT_STATIC int HandleFileOpOperation(
         
         // TODO(#1367): We need to handle failures to lookup the vnode.  If we fail to lookup the vnode
         // it's possible that we'll miss notifications
-        errno_t toErr = vnode_lookup(newPath, 0 /* flags */, &currentVnode, context);
+        errno_t toErr = vnode_lookup(newPath, /* flags: */ VNODE_LOOKUP_NOFOLLOW, &currentVnode, context);
         if (0 != toErr)
         {
-            KextLog_Error("HandleFileOpOperation (KAUTH_FILEOP_RENAME): vnode_lookup failed, errno %d for path '%s'", toErr, newPath);
+            KextLog_Level logLevel = ActiveProvider_FindForPath(newPath) != RootHandle_None ? KEXTLOG_ERROR : KEXTLOG_DEFAULT;
+            KextLog_Printf(logLevel, "HandleFileOpOperation (KAUTH_FILEOP_RENAME): vnode_lookup failed, errno %d for path '%s'", toErr, newPath);
```

Comment:
I'm not sure if we want to drop the log level just yet: perhaps we should leave it at `KEXTLOG_ERROR` for all `vnode_lookup()` failures for now to see if the symlink change improved matters?

Reply:
> perhaps we should leave it at KEXTLOG_ERROR for all vnode_lookup() failures for now to see if the symlink change improved matters?

@pmj I like the idea of leaving it KEXTLOG_ERROR for all vnode_lookup() failures for now.  If we continue to see this message for paths outside of the root we can decide if we want to silence them or root cause the issue (in case the same thing could happen inside the root).